app/controllers/application_controller.rb

```diff
@@ -22,6 +22,17 @@ class ApplicationController < ActionController::Base
   slimmer_template 'wrapper'
 
 protected
+
+  # Set up the navigation helpers. This will make breadcrumbs appear for pages
+  # that have a content item. In the near future this will also provide data to
+  # render the sidebar.
+  def setup_navigation_helpers
+    content_item = content_store.content_item(request.path)
```

Comment:
That is yet to be decided, we're starting mainstream migration discovery today :)